tests/product/test_package_install.py

```diff
@@ -91,91 +91,6 @@ def test_install_exclude_nodes(self):
-                'No such file or directory\n\nAborting.\n'
-        expected = ''
-        for host in self.cluster.all_internal_hosts():
-            expected += error % host
-
-        self.assertEqualIgnoringOrder(cmd_output, expected)
-
-    def test_install_no_path_arg(self):
-        self.installer.copy_presto_rpm_to_master()
-        output = self.run_prestoadmin('package install', raise_error=False)
-        self.assertEqual(output, 'Incorrect number of arguments to task.\n\n'
-                                 'Displaying detailed information for task '
-                                 '\'package install\':\n\n'
-                                 '    Install the rpm package on the cluster\n'
-                                 '    \n    Args:\n'
-                                 '        local_path: Absolute path to the rpm'
-                                 ' to be installed\n        '
-                                 '--nodeps (optional): Flag to indicate if '
-                                 'rpm install\n'
-                                 '            should ignore c'
-                                 'hecking package dependencies. Equivalent\n'
-                                 '            to adding --nodeps flag to rpm '
-                                 '-i.\n\n')
-
-    def test_install_already_uninstalled(self):
-        output = self.run_prestoadmin('package uninstall some_rpm -H %(master)s', raise_error=False)
-        expected = self.escape_for_regex(self.replace_keywords(
-            '\n\nAborting.\nFatal error: [%(master)s] Package is not installed',
-            **self.installer.get_keywords()))
-        self.assertRegexpMatchesLineByLine(
-            output.splitlines(),
-            expected.splitlines()
-        )
-
-    def test_install_already_installed(self):
-        rpm_name = self.installer.copy_presto_rpm_to_master()
-        self.run_prestoadmin('package install /mnt/presto-admin/%(rpm)s -H '
-                             '%(master)s', rpm=rpm_name)
-        self.installer.assert_installed(self, self.cluster.master)
-        cmd_output = self.run_prestoadmin(
-            'package install /mnt/presto-admin/%(rpm)s -H %(master)s',
-            rpm=rpm_name, raise_error=False)
-        expected = self.escape_for_regex(self.replace_keywords("""
-Fatal error: [%(master)s] sudo() received nonzero return code 1 while \
-executing!
-
-Requested: rpm -i /opt/prestoadmin/packages/%(rpm)s
-Executed: sudo -S -p 'sudo password:'  /bin/bash -l -c "rpm -i \
-/opt/prestoadmin/packages/%(rpm)s"
-
-Aborting.
-Deploying rpm on %(master)s...
-Package deployed successfully on: %(master)s
-[%(master)s] out: 	package %(rpm_basename)s is already installed
-[%(master)s] out: """, **self.installer.get_keywords()))
-        self.assertRegexpMatchesLineByLine(
-            cmd_output.splitlines(),
-            expected.splitlines()
-        )
-
-    def test_install_not_an_rpm(self):
```

Comment:
This is also testing the rpm: if we give something other than an rpm file to the rpm utility, we make sure it crashes.